variables.tf

```diff
@@ -192,3 +192,14 @@ variable "perform_proxy_client_setup" {
   )
   default = null
 }
+
+variable "example_valid" {
+  description = "To check if the example is valid."
```

Comment:
what does this mean? Needs better descrption

Reply:
@ocofaigh https://stackoverflow.com/a/66682419 has been used. example_valid variable removed . Thank you